### PR TITLE
Care Leaver exemption criteria satisfied tests

### PR DESCRIPTION
**Pull request by solomonahmed**
## What does this pull request do?

Please describe what you did.
New feature test added for the new care leaver exemption criteria satisfied code LE0001.

## Why make these changes?

Please describe why the changes were needed.
The LE0001 code needs to be available for manually submitted claims and bulk load claims.  The tests ensure changes to add the code and validate it's use meet expectations. 

## Checklist

- [x] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-528

### CODE REVIEW

File: features/check_care_leaver_validation_for_immigration.feature [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  Scenario Outline: Add care leaver Immigration (non-Asylum) claims
+    Given user is on their submission details page
+    When user adds an outcome for Immigration with "<case id>", "<mt>", "<ecs code>", "<ecf ref>", "<case start date>", "<pa>" and "<ap>"
+    Then the outcome saves successfully
+			| case id | mt | ecs code | ecf ref | case start date | pa | ap |
+			| 600 | IMCA:ICZN | LE001 | 1234567AB | 01/11/19 | PA00140 | AP00198 |
+			| 601 | IMCA:IDOM | LE001 | 1234567AB | 01/11/19 | PA00140 | AP00198 |
+			| 602 | IMCA:IEMP | LE001 | 1234567AB | 01/11/19 | PA00140 | AP00198 |
+			| 603 | IMCA:IEUL | LE001 | 1234567AB | 01/11/19 | PA00140 | AP00198 |
+			| 604 | IMCA:IFME | LE001 | 1234567AB | 01/11/19 | PA00140 | AP00198 |
```
Comment (your) [inline features/check_care_leaver_validation_for_immigration.feature]: toooo much indentation? - bad linter, bad! 😭

File: features/check_care_leaver_validation_for_immigration.feature [hunk 2]
```diff
@@ -0,0 +1,82 @@
+Feature: Check Care Leaver Validation for Immigration claims
+
+  In order to submit care leaver claims
+  As a provider
+  I want to use the new care leaver code with an ecf reference
```
Comment (your) [inline features/check_care_leaver_validation_for_immigration.feature]: 💭for the future: I think we should either have a legend for all of these acronyms (ecf, mt1, etc.) or just repeat all the time what they actually mean...

File: features/check_care_leaver_validation_for_immigration.feature (hunk 1 above)
Comment (solomonahmed) [reply features/check_care_leaver_validation_for_immigration.feature]: I'm sure I linted this.  I'll have a look.

File: features/check_care_leaver_validation_for_immigration.feature (hunk 2 above)
Comment (solomonahmed) [reply features/check_care_leaver_validation_for_immigration.feature]: I agree.  Maybe I'll create a legend.txt file in the folder structure somewhere?

**Pull request merged**